src/__tests__/CitiesList.test.js

```diff
@@ -0,0 +1,23 @@
+import { getCurrentState, setData } from '../utils/mockedFunction';
+
+describe('Testing cities list', () => {
+  test('Storing new data into the store ', () => {
+    const payload = {
+      id: 4,
+      name: 'Cairo',
+      population: '12,000,000',
+      country: 'Egypt',
+      timeZone: 'Africa',
+      moreInfo: {
+        cityImage: 'test.png',
+      },
+    };
```

Comment:
- [ ] You only added test for mock utils. kindly add component tests using react testing library and jest.

Reply:
I have added the test for the components.